index.html

```diff
@@ -1,523 +1,1896 @@
-                    </path>
-                </g>
-            </svg>
-                <h2 class="m-3 mr-auto customFooterCopyright">© 2020 Newsweek</h2>
-                <nav class="col-sm-8 col-md-3 ml-auto mr-3 d-flex justify-content-between m-3">
-                    <img class="customSocialIcon" src="images/iconFace.png" alt="alt">
-                    <img class="customSocialIcon" src="images/iconTwitter.png" alt="alt">
-                    <img class="customSocialIcon" src="images/iconInsta.png" alt="alt">
-                    <img class="customSocialIcon" src="images/iconTumbir.png" alt="alt">
-                    <img class="customSocialIcon" src="images/iconLinkedin.png" alt="alt">
-                </nav>
+            <p class="d-inline-block mb-2 customArticleIntro">
+              Culpa eiusmod consectetur ea cillum ex aliquip ad. Irure velit
+              nisi enim consectetur do anim. Cillum aliquip do nostrud velit.
+            </p>
+          </article>
+        </section>
+        <section class="w-100 order-8 p-0 d-flex flex-wrap">
+          <h3 class="customText3">SUBSCRIBE</h3>
+          <article class="col-sm-12 col-lg-6">
+            <div class="m-3 p-0 d-flex justify-content-around flex-wrap">
+              <div class="col-md-6">
+                <img
+                  class="w-50 ml-auto"
+                  src="https://d.newsweek.com/en/full/1607281/cover-july-10-17-2020.jpg?w=170&q=95&f=2207bd7076117dd4ea34f97a3532195f"
+                  alt="alghjkghjkt"
+                />
+                <img
+                  class="w-50 ml-n5 mt-2"
+                  src="https://d.newsweek.com/en/full/1610195/cover-july-24-2020.jpg?w=170&q=95&f=2ea70c30e8f5467c1e61021ae9778138"
+                  alt="alghjkght"
+                />
+              </div>
```

Comment:
You did a great work in this whole section, and we are almost there. However, there are still a few details missing (all screen sizes).

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88578409-e6eb5480-d01e-11ea-9bfa-f756c84483bf.png) | ![image](https://user-images.githubusercontent.com/61479648/88578454-f9658e00-d01e-11ea-9d89-4912695ffb2d.png) |

- [ ] Missing the subscribe button (all screen sizes)

- [ ] Border on the starting 5 card

- [ ] Blue background in the Subscribe

Reply:
1 - The "subscribe" red button is not appearing to me on the original website, but I added as you required.

![image](https://user-images.githubusercontent.com/63410412/88581508-a510dd00-d023-11ea-8905-cb80a2e2ef5b.png)

2 - What are you referring on "Blue background in the Subscribe"?